templates/bashrc.osh-template

```diff
@@ -38,7 +38,8 @@ OSH_THEME="font"
 # Uncomment the following line if you want to change the command execution time
 # stamp shown in the history command output.
 # The optional three formats: "mm/dd/yyyy"|"dd.mm.yyyy"|"yyyy-mm-dd"
-# HIST_STAMPS="mm/dd/yyyy"
+# IF not set, default value is: yyyy-mm-dd
+# HIST_STAMPS="yyyy-mm-dd"
```

Comment:
The default behavior seems to be slightly different from `yyyy-mm-dd`, i.e., there is no brackets `[...]` around the date and time.

Reply:
d830c75 I decided to change not to enclose the date and time within [...] by default. Instead, I have added new possible format names as `[mm/dd/yyyy]`, `[dd.mm.yyyy]`, and `[yyyy-mm-dd]`. Also, the colors are only enabled with the formats with `[]`.